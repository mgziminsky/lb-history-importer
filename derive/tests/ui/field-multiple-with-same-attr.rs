use lb_importer_derive::IntoPayload;

#[derive(IntoPayload)]
struct MultipleError {
    #[track]
    track1: String,
    #[track]
    track2: String,
    other: String,
}

fn main() {}
