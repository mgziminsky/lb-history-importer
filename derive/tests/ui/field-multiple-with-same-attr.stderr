error: Not allowed on multiple fields. Also present on 'track1'
 --> tests/ui/field-multiple-with-same-attr.rs:7:5
  |
7 |     #[track]
  |     ^^^^^^^^

error: Not allowed on multiple fields. Also present on 'track2'
 --> tests/ui/field-multiple-with-same-attr.rs:5:5
  |
5 |     #[track]
  |     ^^^^^^^^
